apps/rellotge/rellotge.js

```diff
@@ -129,51 +129,51 @@
     var m = d.getMinutes();
 
     let t;
-    if (m >= 0 && m < 2) {
+    if (m = 0) {
```

Comment:
for example here:
```suggestion
    if (m == 0) {
```

Reply:
Surely my sight is not as good as it used to be. Didn't see it coming, but now it's corrected.

Reply:
Thanks - easy missed!